### ISSUE TITLE
Example config broken after move to github

### ISSUE THREAD

**Issue by JeremyMcCormick** [iLCSoft/iLCInstall#7]
I can't build my config that used to work fine several weeks ago...

```
$ python ./ilcsoft-install -i ./examples/slic/release-scratch.cfg

********************************************************************************
*** Downloading sources for ILCUTIL version v01-02-01...
********************************************************************************
svn download cmd: svn checkout https://svnsrv.desy.de/public/ilcutil/ilcutil/tags/v01-02-01 v01-02-01
svn: Could not open the requested SVN filesystem
```

Can you please provide me some guidance on how to fix this? 

The DESY SVN URLs that used to function no longer work at all, and I can't build my software because of this.

Thanks.

**Comment by gaede** [comment]
We have moved most of the iLCSoft packages to GitHub. Simply change:
```
ilcsoft.downloadType = "svn"
```
to
```
ilcsoft.downloadType = "GitHub"
```
For the packages that are still in svn, you will have to set the download.root and donwload.type explicitly in the confige file, e.g.:
https://github.com/iLCSoft/iLCInstall/blob/master/releases/HEAD/release-ilcsoft.cfg#L167

**Comment by petricm** [comment]
Hi,
In addition to the comment of @gaede the URL you want to checkout is wrong
```
svn co https://svnsrv.desy.de/public/ilcutil/ilcutil/tags/v01-02-01 v01-02-01
```
instead of this
```
svn co https://svnsrv.desy.de/public/ilctools/ilcutil/tags/v01-02-01 v01-02-01
```

**Comment by JeremyMcCormick** [comment]
Thanks for the advice.  I was able to get my build working again.

The ILCUTIL package config needed to be changed to this:

```
ilcutil = ILCUTIL( ILCUTIL_version )
+ilcutil.download.type="GitHub"
```

I will go ahead and make a fork for merging in the minor config changes.

Another question: Can I set the installation base directory without altering the `versions.py` file?

I saw something about setting `ilcsoft_install_prefix` externally but wasn't sure exactly how to do it.

**Comment by JeremyMcCormick** [comment]
Additional question:

Is this URL going to be left permanently?

```
https://svnsrv.desy.de/public/aidasoft/DD4hep/tags/v00-15/
```

If not I guess I would need to switch over to using the github version.

**Comment by petricm** [comment]
The installation base is set when you do this:
```
ilcsoft = ILCSoft( ilcsoft_install_dir )
````
in the cfg file. The `versions.py` is executed a few lines before that. 

We are not planning to remove the tags from DESY SVN, however we would still recommend the usage of GitHub.

**Comment by gaede** [comment]
You can keep using the _old_ versions from SVN - however all new versions will be only available from GitHub.

**Comment by JeremyMcCormick** [comment]
Resolved by PR #11.
